lib/modules/manager/kustomize/extract.spec.ts

```diff
@@ -595,6 +596,35 @@ describe('modules/manager/kustomize/extract', () => {
     });
   });
 
+  it('parses HelmChartInflationGenerator', () => {
+    const content = codeBlock`
+      apiVersion: builtin
+      kind: HelmChartInflationGenerator
+
+      metadata:
+        name: argocd-chart
+
+      name: argo-cd
+      repo: https://argoproj.github.io/argo-helm
+      version: 6.5.1
+      namespace: argocd
+      releaseName: argocd
+      `;
+    const res = extractPackageFile(content, 'kustomization.yaml', {});
```

Comment:
are you sure about the filename? 🤔 I'm confused. do you have some sample repos?

Reply:
The filename can be anything, there isn't any convention AFAIK

https://github.com/search?q=path%3A*.yaml%20%22kind%3A%20HelmChartInflationGenerator%22&type=code